docs/strategy-customization.md

```diff
@@ -639,6 +639,121 @@ Stoploss values returned from `custom_stoploss` must specify a percentage relati
+
+    Most of the time we do not need power and flexibility offered by `merge_informative_pair()`, therefore we can use a decorator to quicly define informative pairs.
+
+    ``` python
+
+    from datetime import datetime
+    from freqtrade.persistence import Trade
+    from freqtrade.strategy import IStrategy, informative
+
+    class AwesomeStrategy(IStrategy):
+        
+        # This method is not required. 
+        # def informative_pairs(self): ...
+
+        # Define informative upper timeframe for each pair. Available in populate_indicators and 
+        # other methods as 'rsi_1h'.
+        @informative('1h')
+        def populate_indicators_1h(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+            dataframe['rsi'] = ta.RSI(dataframe, timeperiod=14)
+            return dataframe
+
+        # Define BTC/STAKE informative pair. Available in populate_indicators and other methods as
+        # 'btc_rsi_1h'. Current stake currency should be specified as {stake} format variable 
+        # instead of hardcoding actual stake currency. Available in populate_indicators and other 
+        # methods as 'btc_rsi_1h'.
+        @informative('1h', 'BTC/{stake}')
+        def populate_indicators_btc_1h(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+            dataframe['rsi'] = ta.RSI(dataframe, timeperiod=14)
+            return dataframe
+
+        # Define BTC/ETH informative pair. You must specify quote currency if it is different from
+        # stake currency. Available in populate_indicators and other methods as 'eth_btc_rsi_1h'.
+        @informative('1h', 'ETH/BTC')
+        def populate_indicators_eth_btc_1h(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+            dataframe['rsi'] = ta.RSI(dataframe, timeperiod=14)
+            return dataframe
+    
+        # Define BTC/STAKE informative pair. A custom formatter may be specified for formatting
+        # column names. Format string supports these format variables:
+        # * {asset} - full name of the asset, for example 'BTC/USDT'.
+        # * {asset_short} - short name of the asset, for example 'eth' when quote currency matches
+        #   stake currency or 'eth_btc' otherwise.
+        # * {name} - name of dataframe column.
+        # * {timeframe} - timeframe of informative dataframe.
+        # A callable `fmt(**kwargs) -> str` may be specified, to implement custom formatting.
+        # Available in populate_indicators and other methods as 'rsi_upper'.
+        @informative('1h', 'BTC/{stake}', '{name}')
+        def populate_indicators_btc_1h_2(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+            dataframe['rsi_upper'] = ta.RSI(dataframe, timeperiod=14)
+            return dataframe
+    
+        def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+            # Strategy timeframe indicators for current pair.
+            dataframe['rsi'] = ta.RSI(dataframe, timeperiod=14)
```

Comment:
Does the dataframe passed in here already include all the above informative pairs merged to dataframe? 

Are informative pairs worked in sequence (so if i specify 2 methods - can the 2nd method rely on the first?) - or can we not guarantee sequence of the informative pair methods?

Are checks in place that ensure naming uniqueness (no column overwriting) if i'm using custom formats, and cause colisions this way?

Reply:
> Does the dataframe passed in here already include all the above informative pairs merged to dataframe?

Yes, that is the whole point of this decorator ;)

> Are informative pairs worked in sequence (so if i specify 2 methods - can the 2nd method rely on the first?) - or can we not guarantee sequence of the informative pair methods?

No. Informative pairs are processed in isolation. If one informative depends on another - old method should be used.

> Are checks in place that ensure naming uniqueness (no column overwriting) if i'm using custom formats, and cause colisions this way?

No, but pandas does it for us already. When merging dataframes it will append extra bit to columns with colliding names. That wont go unnoticed by the user.

Reply:
i did know the answer to these questions (except for the 2nd one) - but the user will not (hence the comment on the documentation page - not someplace else ;)